testing/src/main/java/org/oppia/android/testing/espresso/TextInputAction.kt

```diff
@@ -0,0 +1,56 @@
+package org.oppia.android.testing.espresso
+
+import android.view.View
+import androidx.test.espresso.matcher.BoundedMatcher
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+
+/**
+ * Action for matching error text and performing other actions on TextInputLayout
+ * in a test infrastructure-specific way.
+ */
+class TextInputAction {
+
+  companion object {
+    /**
+     * Check that the TextInputLayout view has the [expectedErrorText] error text.
+     *
+     * @returns a [BoundedMatcher] that matches the [expectedErrorText] with the
+     * TextInputLayout's error text
+     */
+    fun hasErrorText(expectedErrorText: String): BoundedMatcher<View, TextInputLayout> {
+      return ErrorTextExisted(expectedErrorText)
+    }
+
+    /**
+     * Check that the TextInputLayout view has an empty error text.
+     *
+     * @returns a [BoundedMatcher] that matches if error text is null or empty.
```

Comment:
```suggestion
     * @returns a [BoundedMatcher] that matches if error text is null or empty
```

At-params don't end in punctuation unless they are multiple sentences.

Reply:
Done.